vipps-recurring-api.md

```diff
@@ -161,25 +163,30 @@ For a `"transactionType": "DIRECT_CAPTURE"` setup, the normal flow would be:
 
 ### Reserve capture
 
+**Note:** Reserve capture on recurring charges is available in the recurring API v3 (Coming soon) 
+In the API V2, reserve capture is only available on initial charges.
+
 For a `"transactionType": "RESERVE_CAPTURE"` setup, the normal flow would be:
 
 1. Create a (draft) agreement using the [`draft agreement`][draft-agreement-endpoint] endpoint.
    The user can now confirm the agreement in Vipps (the app). See [Create a new agreement](#create-an-agreement).
 2. The user approves the agreement in Vipps:
-   This will result in a capture of the initial charge (if one was defined in the first step).
-3. Retrieve the (hopefully approved) agreement by calling the [`fetch agreement`][fetch-agreement-endpoint] endpoint.
+   This will result in a capture(or reserve) of the initial charge (if one was defined in the first step).
+   See [Initial charge](#initial-charge).
+3. Retrieve the agreement by calling the [`fetch agreement`][fetch-agreement-endpoint] endpoint.
    See [Retrieve an agreement](#retrieve-an-agreement).
    **Note:** At this point the agreement will be `ACTIVE` if the user completed step 2.
-4. If there is a product that is shipped to the customer, the initial charge should be captured at this point.
-   Capture the charge by calling the [`capture charge`][capture-charge-endpoint] endpoint.
-   If there is no product being shipped, or a need to provide access at a later point - the merchant should change the
-   merchant sale unit setup to use `DIRECT CAPTURE` instead.
-5. For all future charges, you must create a charge by calling the [`create charge`][create-charge-endpoint] endpoint.
+4. All future charges can be created by using the [`create charge`][create-charge-endpoint] endpoint.
+   For reserve capture you must set `"transactionType": "RESERVE_CAPTURE"`.
    See [Create a charge](#create-a-charge).
    Based on the `due` set in the request, we will try to process the charge on that day.
    If for some reason, a charge fails to be processed,
    we will retry for the number of days specified by the `retryDays` value.
    We recommend at least 2 days retry.
+5. If there is a product that is shipped to the customer, the charge should be captured at this point. 
+   Capture the charge by calling the [`capture charge`][capture-charge-endpoint] endpoint.
+   If there is no product being shipped, nor a need to provide access at a later point - the merchant should change the
+   merchant sale unit setup to use `DIRECT CAPTURE` instead and create charge with `"transactionType": "DIRECT_CAPTURE"`.
```

Comment:
Not sure if the sales unit needs to change for recurring. Dont think we check anywhere for this

Reply:
Ok will remove it from documentation